### DISCUSSION TITLE
SASL connectivity options for UserOperator

### DISCUSSION THREAD

**Discussion by palson** [strimzi/strimzi-kafka-operator#5602] [General]
Are there any plans to add the same SASL/SCRAM connectivity options, as for TopicOperator - #5201, for UserOperator?

**Comment by scholzj** [comment]
I guess this is waiting for your contribution ;-).

The main use-case for the User Operator is to work within a Strimzi Kafka cluster. When deployed there, it is always using TLS client authentication. It does not need any SASL support there. So I do not think anyone from the core team plans to add support for it. But as with the Topic Operator, we are always opened to contributions.
